Carbonite.Quests/NxQuest.lua

```diff
@@ -51,6 +51,296 @@ BINDING_NAME_NxWATCHUSEITEM	= L["NxWATCHUSEITEM"]
 
 local IsClassic = select(4, GetBuildInfo()) < 20000
 
+

```

Comment:
Leave Nx.List where it was, thats in Nx.Ui file, this is used in other modules so it shouldnt be moved, becouse if someone doesnt use Quest Module this change ill broke Carbonite.